domains/dhilipan.json

```
@@ -6,7 +6,7 @@
   },
   "record": {
     "A": ["75.2.60.5"],
-    "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
-    "TXT": "v=spf1 include:spf.improvmx.com ~all"
+    "MX": ["mx1.simplelogin.co.", "mx2.simplelogin.co."],
```

Dots aren't necessary for MX records.
```suggestion
    "MX": ["mx1.simplelogin.co", "mx2.simplelogin.co"],
```

---

> Dots aren't necessary for MX records.

Simplelogin docs insist on it.